pylsp/workspace.py

```diff
@@ -76,7 +91,8 @@ def root_uri(self):
         return self._root_uri
 
     def is_local(self):
-        return (self._root_uri_scheme in ['', 'file']) and os.path.exists(self._root_path)
+        return (self._root_uri_scheme in ['', 'file']) and os.path.exists(
+            self._root_path)
```

Comment:
- Please revert this change because the line length limit of this project is 120 columns.
- Also, please refrain of making this kind of changes in the future because they don't add anything significant to your work and make PRs harder to review.

Reply:
f5a2992d57c96e66191a04691f7947b01e28af27 (sorry for running black)